### PR TITLE
feat(ui): In env dropdown menu, add button to manage envs

### PR DESCRIPTION
**Pull request by benvinegar**
image

### CODE REVIEW

**Comment by benvinegar** [comment]
cc @saragilford

File: src/sentry/static/sentry/app/components/projectHeader/index.jsx [hunk 1]
```diff
@@ -107,6 +108,16 @@ class ProjectHeader extends React.Component {
                     {env.displayName}
                   </MenuItem>
                 ))}
+                <MenuItem divider={true} />
+                <div style={{textAlign: 'center', padding: '5px 0px'}}>
+                  <Button
+                    to={`/settings/organization/${org.slug}/project/${project.slug}/environments/`}
```
Comment (lynnagara) [inline src/sentry/static/sentry/app/components/projectHeader/index.jsx]: Let's do `{features.has('new-settings') ? : oldpath : newpath}`

Comment (benvinegar) [reply src/sentry/static/sentry/app/components/projectHeader/index.jsx]: Done.

Comment (lynnagara) [reply src/sentry/static/sentry/app/components/projectHeader/index.jsx]: thanks!

**Review by lynnagara — APPROVED** [review]

**Pull request merged**